cwms-data-api/src/test/java/cwms/cda/helpers/DTOMatch.java

```diff
@@ -57,8 +58,7 @@ public static void assertMatch(CwmsId first, CwmsId second) {
 
     public static void assertMatch(StreamLocation streamLocation, StreamLocation deserialized) {
         assertAll(
-            () -> assertMatch(streamLocation.getId(), deserialized.getId()),
-            () -> assertMatch(streamLocation.getStreamNode(), deserialized.getStreamNode()),
+            () -> assertMatch(streamLocation.getStreamLocationNode(), deserialized.getStreamLocationNode()),
```

Comment:
add message

Reply:
Added messages. (Added them on the primitive comparisons)